sycl/test/esimd/on-device/BitonicSortK.cpp

```diff
@@ -0,0 +1,718 @@
+// TODO enable on WIndows
+// REQUIRES: linux
+// REQUIRES: gpu
+// RUN: %clangxx-esimd -fsycl %s -o %t.out
+// RUN: %ESIMD_RUN_PLACEHOLDER %t.out
```

Comment:
Will it work on host device? If so please add run command. If not add corresponding comment

Reply:
It won't. 

> If so please add run command. If not add corresponding comment

I'd rather avoid adding same explanation into every test. I will add a readme.